src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
Yes we could do it like this, but that will mix two issues, what was identified to be not desired.

The core issue of this normalization preference page is to normalize the loudness of all inputs in a way that they equal, but keeps enough headroom for dynamic peaks of a single track required for classic tracks or non compressed club tracks. 

It is a separate issue to have a mixing headroom for effects and mixing two tracks together. 

I have learned that experienced DJs are aware of this issue and do not need such a mixing headrooms, since they will use the EQ or filter or special xfader characteristic to avoid clipping. Doing this they are able to use the full dynamic range of the sound system.  And that was probably the reason why @rryan  has removed the hidden 6 dB headroom from the gain calculation. 

If you use pop music or compressed radio tracks, practically the default ReplayGain level already offers a mixing headroom. And of cause the headroom set by these boost sliders can be used for dynamic track and for mixing.